### ISSUE TITLE
Tracker QoL improvements

### ISSUE THREAD

**Issue by Kopo942** [AeroScripts/QuestieDev#1012]
After playing around for a while with the new tracker and realizing how accustomed I am to the _all-pretty-n-stuff_ retail quest tracker, I came up with a couple of ~~nitpicky~~ design/config improvements that could be added to Questie at some point in time (Soon-ish™). Or then not.

Adjust(able) padding between quest name and objectives
- 

![](https://i.gyazo.com/d9fbdd6002361ec23a196dc30f8eeb13.png)
At the moment, certain quests that have letters with descenders in the quest name overlap the objective names of said quests (example above). Maybe increase the padding or add adjustable padding as a config option?

Background settings
-
Add config options for 1) toggling the mouseover background and 2) changing its' opacity/color. For some reason, the current mouseover-background style really bothers me and I would rather live without it. <sup>_pls no hate_</sup>

Mouseover quest highlight + quest log shortcut
-
Again, quality of life improvements. Coming from retail, the "Show quest in quest log" shortcut is left-click by default. Could this also be the case in classic? Highlight quest on mouseover would also be a nice addition, would probably help people with impaired vision and make selecting a quest clearer in the first place (GIF below). _EDIT WHILE WRITING THIS: This was requested in #1011. Referencing it here._
![](https://i.gyazo.com/4141c2ce23d3949befa7963fc10f4167.gif)

Toggle showing quest level in tracker / globally
- 
Implementing this would make the quest log a tad bit tidier IMO. It also seems that I'm blind and this was already requested in #995, I'm gonna be a bad boy and still add it here. Yikes.

SHORTCUT FOR ALL THE THINGS!! (quest untracking)
- 
As in, add a shortcut for untracking a quest in the tracker. Shift-leftclick is the default for Retail, maybe do same behaviour as default? _EDIT WHILE WRITING THIS: This was (again..) requested in #1010. Referencing it here. Can you now see it took me almost 45 minutes to write this whole report??_

Redesign right-click menu (??)
- 
Don't really know how to explain this. The right-click menu just feels a bit messy at the moment. Or then I'm dumb.

Tracker anchoring is wonky (bug?)
- 
Kinda discovered this by accident. The anchoring of the tracker behaves differently depending on the position of the tracker (see below GIFs). In the upper one, anchoring behaves "as expected" instead of jumping around (lower GIF). _EDIT: #1003_
![](https://i.gyazo.com/dfcb240567d3d19289ee958e0dba3b68.gif)
![](https://i.gyazo.com/9614da5d4786da645b6340cf68dea921.gif)

Woah. This became a much lengthier write-up than I initially thought. Might split this into different issues if it becomes a problem later on. Hope you enjoy <3

**Comment by BreakBB** [comment]
Thank you very, very much for this detailed report just hours after we've finally released the tracker.
So yes, we're trying to make all/most shortcuts configurable. Some can already be changed in the Tracker tab.

The padding of quests should be fixed really soon, not sure if configurable makes much sense :thinking: 

Adding the questlevel to the questlog is already on our list as you already stated. We're looking into a way to make it something like (60), (60+), (60D) and (60R). For normal, elite, dungeon and raid quests.

Since the tracker has just been released there will be changes to it in future updates to make it even better. If you have specific ideas we're happy to hear them :raised_hands: 

And about your anchor problem: Yes, this is weired. We've tried to make the tracker a bit smart so the tracker will always display it's quest on the screen and not grow over it. That seems to be the reason your wobbeling tracker.

**Comment by MysticalOS** [comment]
Main improvement I want to see, is quests not being untracked unless I untrack them. That's literally most triggering "feature" of vanilla/classic. Modern quest watch fixes that but I don't really want to run both mods. That's something I'd love to see questie fix as well.

**Comment by BreakBB** [comment]
@Kopo942 If you've followed the development of Questie in the past weeks we've mostly focused on bug fixes and stability improvements.

But with the last updates we also added many new features. For this issue some suggestions have already been implemented and released.

The tracker is becoming better and better. We're planning to add/fix that background highlighting for the next release as well.

Included by now is:
- Shift + leftclick (as default) to untrack a quest
- The anchoring should be fixed
- Questlevel can be toggled

Moreover we're adding some color to the right click menu and will have a look at the padding as well. Highlighting individual quests will be added soon :tm: as well, but might take some more time.
